content/en/docs/concepts/services-networking/service.md

```diff
@@ -870,6 +879,40 @@ in those modified security groups.
 
 {{< /note >}}
 
+#### Other CLB annotations on TKE
+
+There are other annotations for managing Cloud Load Balancer on TKE as described below.
+
+```yaml
+    metadata:
+      name: my-service
+      annotations:
+        service.kubernetes.io/qcloud-loadbalancer-backends-label： `key in (value1, value2)`
+        # Bind Loadbalancers with speicfied nodes
```

Comment:
the convention is to have comments BEFORE the actual stanza.

Reply:
Does this suggest that the annotation description should be placed before the annotation?

The format here is a reference to AWS.
https://kubernetes.io/docs/concepts/services-networking/service/#other-elb-annotations

Reply:
> the convention is to have comments BEFORE the actual stanza.

Does this suggest that the annotation description should be placed before the annotation?

The format here is a reference to AWS.
https://kubernetes.io/docs/concepts/services-networking/service/#other-elb-annotations